packages/vapor/scss/controls/filter-box.scss

```diff
@@ -62,6 +62,10 @@
         padding-right: 40px;
         padding-left: 20px;
 
+        &:focus {
+            outline-color: var(--filter-box-secondary-focus-border-color);
```

Comment:
```suggestion
            outline-color: var(--green);
```

Reply:
I think we could use `var(--green)` directly here instead of creating a new CSS variable. After all, we will do some cleanup on lots of CSS variables made for light reskin.